assembl/static2/js/app/components/administration/discussion/languageAdmin.jsx

```diff
@@ -22,13 +22,31 @@ class LanguageSection extends React.Component {
 
   render() {
     const {i18n, selectedLocale, data } = this.props;
-    console.log(data);
+
+    const allLangs = {};
+    data.defaultPreferences.languages.forEach((lang) => {
+      allLangs[lang.locale] = {state: false, name: lang.name};
```

Comment:
state is probably not the most meaningful name, given it stores a boolean and given its use below. I would prefer "active" or something like this.